modules/apps/announcements/announcements-web/src/main/resources/META-INF/resources/announcements_admin/js/AnnouncementsManagementToolbarPropsTransformer.js

```diff
@@ -20,7 +20,7 @@ export default function propsTransformer({
 	return {
 		...props,
 		onActionButtonClick(event, {item}) {
-			if (item.data?.action === 'deleteEntries') {
+			if (item?.data?.action === 'deleteEntries') {
```

Comment:
I think it was to be defensive, but when I was going through the code where there are cases where we added the optional chaining to `item` and other cases where we were not, but there was no difference in the usage/call of `item`.  So just to be consistent I figured it would be good to pick one or the other and I opted to be "safer" and added them in everywhere.